src/snowflake/snowpark/modin/plugin/extensions/base_overrides.py

```diff
@@ -303,3 +424,1901 @@ def truncate(
 @register_base_not_implemented()
 def __finalize__(self, other, method=None, **kwargs):
     pass  # pragma: no cover
+
+
+# === OVERRIDDEN METHODS ===
+# The below methods have their frontend implementations overridden compared to the version present
+# in base.py. This is usually for one of the following reasons:
+# 1. The underlying QC interface used differs from that of modin. Notably, this applies to aggregate
+#    and binary operations; further work is needed to refactor either our implementation or upstream
+#    modin's implementation.
+# 2. Modin performs extra validation queries that perform extra SQL queries. Some of these are already
+#    fixed on main; see https://github.com/modin-project/modin/issues/7340 for details.
+# 3. Upstream Modin defaults to pandas for some edge cases. Defaulting to pandas at the query compiler
+#    layer is acceptable because we can force the method to raise NotImplementedError, but if a method
+#    defaults at the frontend, Modin raises a warning and performs the operation by coercing the
+#    dataset to a native pandas object. Removing these is tracked by
+#    https://github.com/modin-project/modin/issues/7104
+# 4. Snowpark pandas uses different default arguments from modin. This occurs if some parameters are
+#    only partially supported (like `numeric_only=True` for `skew`), but this behavior should likewise
+#    be revisited.
+
+# `aggregate` for axis=1 is performed as a call to `BasePandasDataset.apply` in upstream Modin,
+# which is unacceptable for Snowpark pandas. Upstream Modin should be changed to allow the query
+# compiler or a different layer to control dispatch.
+@register_base_override("aggregate")
+def aggregate(
```

Comment:
I assume those are directly copied from the base.py. Or is there any difference compare with the original file? if yes, can you mark it here?